plugins/lookup/kustomize.py

```diff
@@ -141,7 +155,21 @@ def run(
         if enable_helm:
             command += ["--enable-helm"]
 
-        (ret, out, err) = run_command(command)
+        environ = None
+        if environment:
+            environ = os.environ.copy()
+            if isinstance(environment, str):
+                if not all(env.count("=") == 1 for env in environment.split(" ")):
+                    raise AnsibleLookupError(
+                        "environment should be dict or string in the format key=value"
+                    )
+                for env in environment.split(" "):
```

Comment:
Please add to the definition that a space should separate multiple key/value elements

Reply:
done

Reply:
the plugin documentation has not been updated